packages/backend-for-frontend/src/model/errors.ts

```diff
@@ -1,16 +1,17 @@
 import { constants } from "http2";
 import {
   ApiError,
-  parseErrorMessage,
-  makeApiProblemBuilder,
   AttributeId,
+  EServiceTemplateId,
+  makeApiProblemBuilder,
+  parseErrorMessage,
 } from "pagopa-interop-models";
 
 export const errorCodes = {
   purposeNotFound: "0001",
   userNotFound: "0002",
   selfcareEntityNotFilled: "0003",
-  descriptorNotFound: "0004",
+  eserviceIsNotDraft: "0004",
```

Comment:
Is this change correct?

Reply:
`descriptorNotFound` is unused, so I just remove it and replaced.